### DISCUSSION TITLE
[Question] How to use assets_dir

### DISCUSSION THREAD

**Discussion by efermon** [flet-dev/flet#3103] [Q&A]
### Question

Hello everyone,
I just discovered Flet and I think it's great. I immediately started a tutorial, but soon ran into the first problem.

Suppose I want the resources folder to be named **assets_04** instead of **assets.** If I run the application by _python app.py,_ it works fine, but if I run the application by _flet run app.py_ it does not work.

Regardless of what you have in asset_dir, flet always looks for the resources in the assets folder

### Code sample

```python
import flet as ft
 
def main(pagina: ft.Page):
    pagina.fonts = {
        'Kanit':'fonts/Kanit-Bold.ttf',
        'MisionEspacial':'fonts/SpaceMission-rgyw9.otf',
    }
   
    pagina.add(
            ft.Text(value='Texto de prueba para verificar fuente 1', font_family = 'MisionEspacial', size=20),
            ft.Text(value='Texto de prueba para verificar fuente 2', font_family = 'Kanit', size=20)
    )

ft.app(main, assets_dir="assets_04")
```

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment]
Can you trying specifying the assets dir through the command line and report why happens? ([Docs](https://flet.dev/docs/reference/cli/run#:~:text=%2Da%20assets_dir%2C%20%2D%2Dassets%20assets_dir))

Also, did you mean you have both directories (assets and assets04) in the root of your app?

**Comment by efermon** [comment]
Thanks for answering.
I've just started a Flet course and I was trying to have an assets
directory per chapter (assets_04 would be the resources directory for
chapter 4).

In the attached image you can see that I only have one assets folder,
assets_04

Right now I just verified that if I specify assets_dir in the execution it
also works fine:
python fonts.py OK
flet run sources.py -a assets_04 OK
flet run fonts.py Doesn't work

I remember that in the code I do specify assets_dir:
ft.app(main, assets_dir="assets_04")

(Sorry if my English is not very good. It is a Google translation

**Comment by efermon** [comment]
It seems that the image did not upload. The tree is:
root
       - folder  assets_04
              - fonts
                  .  file   Kanit-Bold.ttf
                  .  file  SpaceMission-rgyw9.otf . fonts.py
       . file fuentes.py
